src/PokemonContainer.js

```diff
@@ -40,59 +40,59 @@ class PokemonContainer extends Component {
     this.setState({ data: data.results, loading: false, lastPage });
   };
 
-  updateQuery = (event)=>{
-   this.setState({ query: event.target.value });
-}
-  updateSort = (event)=>{
-    this.setState({sortOrder: event.target.value });
+  updateQuery = (event) => {
+    this.setState({ query: event.target.value });
+  }
+  updateSort = (event) => {
+    this.setState({ sortOrder: event.target.value });
   }
 
-  nextPage = async () =>{
-      await this.setState({ page: this.state.page + 1});
-      this.fetchData();
+  nextPage = async () => {
+    await this.setState({ page: this.state.page + 1 });
+    this.fetchData();
   };
 
-  prevPage = async () =>{
-    await this.setState({ page: this.state.page - 1});
+  prevPage = async () => {
+    await this.setState({ page: this.state.page - 1 });
     this.fetchData();
-};
+  };
 
-goToLast = async () => {
+  goToLast = async () => {
     await this.setState({ page: this.state.lastPage });
     this.fetchData();
-};
+  };
 
-searchPokemon = async()=>{
-    await this.setState({page: 1})
+  searchPokemon = async () => {
+    await this.setState({ page: 1 })
     this.fetchData();
-};
+  };
 
-  render() { 
+  render() {
     const { loading, sortOrder } = this.state;
-    return(
+    return (
       <>
         <h1>Pokemon!!</h1>
         <div className="search-controls">
-            <select defaultValue={sortOrder} onChange={this.updateSort}>
-              <option value="asc">Ascending</option>
-              <option value="desc">Descending</option>
-            </select>
-            <input onChange={this.updateQuery} type="text"></input>
-            <button onClick={this.searchPokemon}>Search!</button>
+          <select defaultValue={sortOrder} onChange={this.updateSort}>
+            <option value="asc">Ascending</option>
+            <option value="desc">Descending</option>
+          </select>
+          <input onChange={this.updateQuery} type="text"></input>
+          <button onClick={this.searchPokemon}>Search!</button>
         </div>
         <div className="page-controls">
-            {this.state.page > 1 &&  (
+          {this.state.page > 1 && (
             <button onClick={this.prevPage}>Prev</button>
-            )}
-            {this.state.page < this.state.lastPage && (
+          )}
+          {this.state.page < this.state.lastPage && (
             <>
-                <button onClick={this.nextPage}>Next</button>
-                <button onClick={this.goToLast}>Last page</button>
+              <button onClick={this.nextPage}>Next</button>
+              <button onClick={this.goToLast}>Last page</button>
             </>
-            )}
+          )}
         </div>
-            {this.state.loading && <h3> LOADING!! </h3>}
-        {!this.state.loading && (
+        {loading && <h3> LOADING!! </h3>}
```

Comment:
Remember this should be an actual loading spinner animation rather than simply words saying "loading". It provides a much better UX for the user.